src/tasks/tasks.py

```diff
@@ -74,13 +73,21 @@ def process_single_pair_task_split(split, indexers, is_pair=True, classification
     Returns:
         - instances (Iterable[Instance]): an iterable of AllenNLP Instances with fields
     '''
+    # check here if using bert to avoid passing model info to tasks
+    bert_pair = any(["bert" in idx_name for idx_name in indexers])
```

Comment:
nit: can this check for a stronger condition, or pass an explicit `use_bert` flag to this function? (I worry that you could end up with a really difficult-to-find error if some label namespace ends up having 'bert' as a substring)